### ISSUE TITLE
Upgrade to DataCite Metadata Schema 4.0

### ISSUE THREAD

**Issue by kbraak** [gbif/ipt#1280]
Blocked by [change to gbif-doi](http://dev.gbif.org/issues/browse/POR-3165). 

[Changes in 4.0](http://schema.labs.datacite.org/meta/kernel-4.0/) include:
- Changing resourceTypeGeneral from optional to mandatory
- _Addition of a new property: FundingReference, with subproperties funderName, funderIdentifier, awardNumber, awardURI and awardTitle. Deprecation of contributorType “funder”_
- Addition of new optional subproperties for creatorName and contributorName: familyName and givenName
- Addition of a new relatedIdentifierType option “IGSN”
- Addition of a new subproperty for GeoLocation “geoLocationPolygon”, and changing the definition of the existing GeoLocation subproperties (geoLocationPoint and geoLocationBox)

[According to DataCite](https://blog.datacite.org/metadata-schema-4-webinar/), "These changes are important in improving data discoverability and reusability, and helps to grant proper credit to data creators."

**Comment by MattBlissett** [comment]
For review by @mike-podolskiy90, but I think this is probably already done.

**Comment by mike-podolskiy90** [comment]
@MattBlissett yes, we already use v4
